benchmarks/nemo/nemolite2d/manual_versions/psykal_kokkos/Makefile

```diff
@@ -10,23 +10,56 @@
 # export OMPFLAGS=" -fopenmp"
 # export AR=ar
 
-# Additionally it needs the path to a Kokkos installation and execution
-# devices
-KOKKOS_PATH ?= ${HOME}/build/kokkos
-KOKKOS_DEVICES=OpenMP  # Currently this can only be OpenMP
+# Additionally it needs the path to the Kokkos source code
+KOKKOS_PATH ?= $(HOME)/tools/kokkos
+KOKKOS_DEBUG ?= yes
+
+# If no KOKKOS_DEVICES is specified, by default use the OpenMP
+ifndef $(KOKKOS_DEVICES)
+KOKKOS_DEVICES=OpenMP
+endif
+
+# Configuration specific to the selected Kokkos device 
+ifeq ($(KOKKOS_DEVICES),OpenMP)
+$(info "Using OpenMP device")
+# KOKKOS_OPTIONS := "aggresive_vectorization"
+else ifeq ($(KOKKOS_DEVICES),Serial)
+$(info "Using Serial device")
+# KOKKOS_OPTIONS := "aggresive_vectorization"
+else ifeq ($(KOKKOS_DEVICES),Pthreads)
+$(error "Pthreads device is currently not supported")
+else ifeq ($(KOKKOS_DEVICES),Cuda)
```

Comment:
I would prefer "CUDA" as that's how NVIDIA use it. However, if this is Kokkos' choice rather than yours then obviously there's not a lot we can do about it.

Reply:
This variable is used by Kokkos build system and it expect this specific capitalization.